packages/common.vm/common.vm.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>common.vm</id>
-    <version>3.0.1</version>
+    <version>0.0.0.20220113</version>
```

Comment:
I would prefer to merge https://github.com/mandiant/VM-Packages/pull/21 before this PR as it changes many files and to ensure this PR doesn't break the linter (if we can make that PR work 🤞). Then we should increase this (to today's date) to avoid conflicts with that PR (otherwise this PR won't push the package as it has the same version.